pom.xml

```diff
@@ -238,6 +238,19 @@
             <version>3.3.3</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.maven.shared</groupId>
+            <artifactId>maven-verifier</artifactId>
+            <version>1.6</version>
```

Comment:
as a common pattern we have followed shall we move these versions as properties ?